### ISSUE TITLE
Cannot Update or Delete Personal App from Teams

### ISSUE THREAD

**Issue by rwoehr** [SharePoint/sp-dev-docs#5750]
## Category

- [x ] Question
- [ ] Typo
- [ ] Additional article idea

## Question
I have been developing a SPFx web part - custom Teams Personal App recently.  I have deployed the personal app to the App Catalog of my tenant several times & have uploaded the manually created manifest several times using "Upload a Custom App" from within Teams.  I am now at a point where I can neither Update or Delete my deployed personal app from Teams.  In fact, I can't even Add the app from within Teams any longer.  I get a "Something went wrong" issue".  

If I try to re-upload the app, I get:

2020-05-15T21:23:22.251Z Tenant app definition already exists. Error: App already exists

If I try to Update the app, I get:

2020-05-15T22:06:55.705Z Failed to update tenant app package. Error: {"data":{"errorCode":"AppDefinitionDoesntExist","message":"App doesn't exist in the tenant. UserId: 'ade85de8-df98-4457-a6f2-78a7c04ce90d', TenantId: '71794340-c78e-4299-b60b-17d55266d8be', AppId: '862b9d78-16da-4c24-8f49-e60021cd7dca'"},"status":404,"config":{"method":"PUT","transformRequest":[null],"transformResponse":[null],"url":"https://teams.microsoft.com/api/mt/part/amer-02/beta/tenant/apps/definitions/862b9d78-16da-4c24-8f49-e60021cd7dca/appPackage","data":{},"headers":{"Authorization":"pii","X-Skypetoken":"pii","Content-Type":"application/zip","X-Client-UI-Language":"en-us","Accept":"application/json, text/plain, */*","X-RingOverride":"ring3_6","x-ms-scenario-id":"511","x-ms-request-id":"ef7d36a32651ce166544bfb6911534f0_512","x-ms-user-type":"user","x-ms-client-type":"web","x-ms-client-env":"msteams-web-prod-usce2-01","x-ms-client-version":"1415/1.0.0.2020051106","x-ms-client-cpm":"ApplicationLaunch","x-ms-session-id":"ef7d36a3-2651-ce16-6544-bfb6911534f0"},"counter":"apps_definitions_PUT","geoRoutingLookupDuration":0.014999997802078724,"decrementNetworkRequests":null,"hostName":"teams.microsoft.com","requestCreationDuration":0.13000000081956387,"requestStartTime":3145153.0600000117,"responseStartTime":3148729.4350000154,"requestDuration":3576.394999981858},"statusText":""}

If I try to Delete the app, I get:

2020-05-15T22:07:56.481Z ChannelService: Team id undefined was not found
angular.min.js:113 
2020-05-15T22:08:00.552Z ngClickDecorator: value not found in enum type. value: 
Then the same "App doesn't exist in the tenant." message as above when Updating.

Is there another was to remove the a custom Personal App from Teams in order to start over clean again?

## Environment details (*development & target environment*)

- **OS**: Windows 10
- **Target Environment**: SharePoint Online
- **Framework**: React 16
- **Browser(s)**: Chrome v81
- **Tooling**: VS Code & SPFx v1.10.0
- **Additional details**:

**Comment by andrewconnell** [comment]
Appears to be dupe of an issue that was just resolved. Please confirm dupe of #5686

**Comment by andrewconnell** [comment]
What do you see if you look at the list of apps installed in your tenant from the Teams Admin Center?

**Comment by rwoehr** [comment]
I have experienced this issue in 2 different tenants.

I went into my development tenant->Teams (Web Version)->Apps->Built for MyTenant->Clicked the ellipsis on the personal app for the context menu & was able to successfully delete the app!  Thank you for fixing this!  I go into Teams Admin Center, search for my app, & see the ability to Update & Delete there as well.  I did not try anything here since the issue is resolved "in this tenant".

Tenant #2 (the company I work for ) - In Teams->Apps->Built for MyTenant->Clicked on the ellipsis on the personal app & the context menu on had the "Copy link" option.  Is this a permissions setting?  I asked the global admin to go into the Teams Admin Center & he was successfully able to delete the app.  After this I was able to successfully redeploy the app.  

I'm in good shape now.  I believe my issue is resolved in both tenants.  Thanks for your help!
